torch/csrc/api/include/torch/nn/functional/vision.h

```diff
@@ -47,52 +47,70 @@ inline Tensor affine_grid(
   return torch::affine_grid_generator(theta, size, align_corners);
 }
 
+// ============================================================================
+
+namespace detail {
 inline Tensor grid_sample(
     const Tensor& input,
     const Tensor& grid,
-    GridSampleOptions options = {}) {
+    std::string mode = "bilinear",
```

Comment:
Are we going to change it to torch:: enums?

Reply:
Yes I will do it in a follow-up PR (before the release)